Dockerfile

```diff
@@ -1,16 +1,23 @@
-FROM golang:1.17.3-alpine3.13 AS build
-WORKDIR /go/src/github.com/jpillora
+FROM golang:1.24-alpine3.20 AS build
+WORKDIR /go/src/
 
 RUN apk update
 RUN apk add git
```

Comment:
I think you could make this `apk --no-cache` and skip the update altogether, if you wanted.